.travis.yml

```diff
@@ -13,7 +13,7 @@ install: true
 
 before_script:
   - if [ "$TRAVIS_JDK_VERSION" == "openjdk8" ]; then
-      export IMAGE_NAME="openjdk:8u212-jre-alpine";
+      export IMAGE_NAME="openjdk:8-jre-slim";
```

Comment:
This file is currently deprecated.
Note that our base image uses openjdk:8u342

https://github.com/seata/seata/blob/7e40e05c1ba4152798aadf1530d3cd6ee9deaa9e/.github/workflows/publish-docker.yml#L44

Reply:
ok.i will check it out.